korman/exporter/animation.py

```diff
@@ -89,21 +93,29 @@ def get_ranges(*args, **kwargs):
                 if i is not None:
                     yield i.frame_range[index]
         atcanim.name = "(Entire Animation)"
-        atcanim.start = self._convert_frame_time(min(get_ranges(obj_action, data_action, index=0)))
-        atcanim.end = self._convert_frame_time(max(get_ranges(obj_action, data_action, index=1)))
+        if anim_data.obj_sdl_anim:
+            atcanim.start = self._convert_frame_time(min(get_ranges(obj_action, index=0)))
+            atcanim.end = self._convert_frame_time(max(get_ranges(obj_action, index=1)))
+        elif anim_data.obj_sdl_anim is None:
```

Comment:
This will cause things like lamp color animations to not have the proper length in age global animations. While I do think `get_ranges` is a bit aggressive, I'm not sure this is the correct fix.

Reply:
Yeah, that was part of my hatchet job of trying to get ATC and global separate. I'll try adding the data_action back in.